README.md

```diff
@@ -3,4 +3,37 @@ Kubernetes Artifacts for
 Container-based Deployment Patterns
 of WSO2 Enterprise Integrator
 
-This project is still under development.
\ No newline at end of file
+This initial version (V0.1) contains the deployment of a single integrator container instance 
```

Comment:
I think it's better not to mention the version here. Release note might be the best place to mention this.